src/GitReleaseManager.Core/Commands/InitCommand.cs

```diff
@@ -4,7 +4,6 @@
 using GitReleaseManager.Core.Configuration;
 using GitReleaseManager.Core.Helpers;
 using GitReleaseManager.Core.Options;
-using GitReleaseManager.Core.Templates;
```

Comment:
This change needs to be reverted.  I believe you ran `dotnet format` before running `build.ps1`, which meant that the T4 template in the project hadn't been generated yet, so this using was seen as not being needed, when actually it is.  This is why the CI build is failing.